chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb

```diff
@@ -68,3 +70,21 @@
     new_alias: new_alias
   )
 end
+
+# This script shuts down non-essential services on the nodes
+# It leaves only database (so we can create a dump of it)
+# and services necessary for managing network traffic of running instances.
+
+# Find out now if we have HA setup and pass that info to the script
+use_ha = !search(:node, "pacemaker_founder:true").empty?
```

Comment:
It does not use the node for anything, it tries to find out if we are using HA setup.

But maybe this should be node specific...